apps/pages/models.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.db import models
+
+from ckeditor.fields import RichTextField
+from mptt.models import MPTTModel, TreeForeignKey
+
+
+class Category(MPTTModel):
+    title = models.CharField(max_length=64, unique=True)
+    slug = models.SlugField(max_length=64, unique=True)
+    description = models.TextField(blank=True)
+    parent = TreeForeignKey(
+        'self',
+        null=True,
+        blank=True,
+        related_name='children',
+        db_index=True,
+        limit_choices_to=models.Q(level=0) | models.Q(level=1)
+    )
+
+    class Meta:
+        verbose_name_plural = 'Categories'
+        ordering = ('id',)
+
+    def __str__(self):
+        return self.title
+
+
+class Page(models.Model):
+    title = models.CharField(max_length=256)
+    content = RichTextField(
+        blank=True,
+    )
+    author = models.CharField(max_length=256, blank=True)
+    url = models.CharField(max_length=128, db_index=True)
```

Comment:
Add a validator to this similar to flatpages, otherwise users can add URLs which don't start with `/`, or end with `/`, etc.